packages/core/src/components/Alert/Alert.jsx

```diff
@@ -18,7 +31,11 @@ export class Alert extends React.PureComponent {
     );
 
     return (
-      <div className={classes} role={this.props.role}>
+      <div
+        className={classes}
+        role={this.props.role}
+        aria-labelledby={this.props.ariaLabelledBy}
```

Comment:
Let's have the aria-labelledby reference the same string as the heading:

`aria-labelledby={this.headingId}`